contracts/mixins/InterestHandler.sol

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2017-2021, bZxDao. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+pragma solidity 0.5.17;
+
+import "../core/State.sol";
+import "../interfaces/ILoanPool.sol";
+import "../utils/MathUtil.sol";
+
+
+contract InterestHandler is State {
```

Comment:
@Drypto13 @eugenesh1 try to find a way where we can store improper values interest rate model